modules/twinkleblock.js

```diff
@@ -984,13 +983,52 @@ Twinkle.block.blockPresetsInfo = {
 		reason: '{{zombie proxy}}',
 		sig: null
 	},
+
+	// Begin partial block templates, accessed in Twinkle.block.blockGroupsPartial
 	'uw-pblock': {
 		autoblock: true,
 		expiry: '24 hours',
 		nocreate: false,
 		pageParam: false,
 		reasonParam: true,
 		summary: 'You have been [[WP:PB|partially blocked]] from certain areas of the encyclopedia'
```

Comment:
partially blocked -> blocked
+"editing" certain areas

Reply:
Here (and for the indef), I'm not sure about this: it's generic so someone could use it to apply non-editing PBs.  This is the edit summary for the template, so if the sysop didn't apply an editing block, they couldn't change the summary.

Reply:
That's true, I would've guessed they can opt for the other presets but as a generic option, the current one fits best, true.